### PR TITLE
Added support email to the error message on zip submissions download

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
None

#### What's this PR do?
Increase poll size of timer task. Which checks if zip file is ready or not via an api

#### How should this be manually tested?
- Download all submission, sga should saw file is ready for download

@pdpinch

### CODE REVIEW

**Comment by pdpinch** [comment]
We need to improve the error message so the user who gets this has some idea what to do. We wasted a lot of time tracking down this error. 

Given the nature of the error -- the zip file creation never responds -- I think all we can do is suggest the user contact the support email address. 

Perhaps "The download file was file not created. Please try again, or contact [support email]"

**Comment by amir-qayyum-khan** [comment]
Done @pdpinch

File: edx_sga/sga.py [hunk 1]
```diff
@@ -572,6 +572,7 @@ def student_view(self, context=None):
             "student_state": json.dumps(self.student_state()),
             "id": self.location.name.replace('.', '_'),
             "max_file_size": self.student_upload_max_size(),
+            "support_email": settings.TECH_SUPPORT_EMAIL
```
Comment (noisecapella) [inline edx_sga/sga.py]: Can you add a unit test to assert this?

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella

**Pull request merged**